Aquarius/Aquarius.cpp

```diff
@@ -30,6 +33,12 @@ int Test::testMain()
     AQ_INFO("Testing %v %v", "client", "logger");
     AQ_CORE_INFO("Testing %v %v", "core", "logger");
 
+    aqAssert(5 == 4);
+    aqAssert(4 == 5, "This is obviously %v", "wrong");
+
+    aqCoreAssert(true == false, "This is obviously false");
+    aqCoreAssert("hello" == "world", "This is obviously %v", "wrong");
+
```

Comment:
After testing, execution is not stopped or paused after this assertion, can we somehow do that?

Reply:
Yeah so now `std::abort()` is used, which is what the assert macro uses to terminate program execution